helm/idol-qms/values.yaml

```diff
@@ -123,8 +122,5 @@ single-content:
   # -- content port service will serve ACI connections on
   aciPort: *queryServiceACIPort
 
-  # -- the content engine's query service name
-  queryserviceName: *queryServiceName
-
```

Comment:
Revert this so the single-content (if deployed) still generates an `idol-query-service`

Reply:
https://github.com/opentext-idol/idol-containers-toolkit/pull/27/commits/37e0deda98d7817e290b34165b85aaaaa6a55593